packages/language-server/src/server.ts

```diff
@@ -63,6 +64,22 @@ connection.onCompletionResolve(
   }
 );
 
+documents.onDidChangeContent(async changeEvent => {
+  // TODO: can we ensure the promise has always been initialized first (never be undefined)?
```

Comment:
@tal-sapan  could you look at the TODO comments with questions here?

Reply:
It is always initialized because `onInitialized` happens first but there is no way to let typescript know that

Reply:
o.k. I will remove the comment and we will leave the defensive if condition in the code